src/dotnet/APIView/APIViewWeb/Managers/APIRevisionsManager.cs

```diff
@@ -828,5 +828,15 @@ private async Task GenerateAPIRevisionInExternalResource(ReviewListItemModel rev
             }
             return result;
         }
+
+        public async Task<APIRevisionListItemModel> UpdateRevisionPackageVersionAsync(APIRevisionListItemModel revision, string packageVersion)
+        {
+            if (packageVersion != null && !packageVersion.Equals(revision.Files[0].PackageVersion))
```

Comment:
just making sure, does `revision.Files[0]` always exists? or would it be better to check and avoid a null pointer

Reply:
It should always exists for a revision. Otherwise a revision is does not have anything to show in APIView.

Reply:
One eye is twitching bc this is a function that anyone can call and I don't want people to get cunexpected behavior. But I understand that it doesn't make sense and it should be good